packages/orbit-components/src/utils/common/index.ts

```diff
@@ -1,9 +1,31 @@
 import type { FlattenSimpleInterpolation } from "styled-components";
 import { css } from "styled-components";
 
+import type { ObjectProperty } from "../../common/types";
+
 export const firstToUpper = (str: string): string => str.charAt(0).toUpperCase() + str.slice(1);
 
 export const defaultFocus = (): FlattenSimpleInterpolation => css`
   outline: 2px solid Highlight;
   outline: 2px solid -webkit-focus-ring-color;
 `;
+
+export const marginUtility = (
+  margin?: React.CSSProperties["margin"] | ObjectProperty,
+): FlattenSimpleInterpolation | null => {
+  if (!margin) return null;
+  if (typeof margin === "string" || typeof margin === "number") {
+    return css`
+      margin: ${margin};
```

Comment:
I noticed a problem here. If we use it like (on Card, for example):
`<Card margin="2" … />`
or
`<Card margin={2} … />`
it will produce this
<img width="307" alt="image" src="https://user-images.githubusercontent.com/6265045/206217011-51bc017b-3619-4298-b823-8a5182e83c07.png">

The same happens with the object. So we have to treat the input, at least, when it is a number and add the "px".
If it is a string I think we can disregard any treatment.